content/docs/2.15/concepts/scaling-jobs.md

```diff
@@ -3,10 +3,12 @@ title = "Scaling Jobs"
 weight = 300
 +++
 
+This page describes the job scaling behavior of KEDA. See the [Scaled Job specification](../reference/scaledjob-spec.md) for details on how to set the behaviors described below.
```

Comment:
This page describes the job scaling behavior of KEDA. 

## Specification

See the [Scaled Job specification](../reference/scaledjob-spec.md) for details on how to set the behaviors described below.